lib/oli_web/components/delivery/students/students.ex

```diff
@@ -375,16 +388,70 @@ defmodule OliWeb.Components.Delivery.Students do
       <% end %>
       <input name="role" value={@add_enrollments_selected_role} />
       <input name="section_slug" value={@section_slug} />
+      <input name="inviter" value={@inviter} />
       <button type="submit" class="hidden" />
     </.form>
     <div class="px-4">
       <p>
         Are you sure you want to enroll <%= "#{if length(@add_enrollments_emails) == 1, do: "one user", else: "#{length(@add_enrollments_emails)} users"}" %>?
       </p>
+      <.render_inviter
```

Comment:
Supernit, but in general we don't name components with a `render_` prefix or similar, since it's implicit that this call renders its content. I think just `inviter` is clear enough 😄

Reply:
Thanks! It's nice to know we have a convention.